meeting/meeting_january_11_2022.md

```diff
@@ -0,0 +1,62 @@
+# January 11th (7 PM CST)/10th (9 AM JST) Meeting 2022
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Chris)**
+* **#Query(Matt)**
+* **#Query(Tibo)**
+* **#Query(Jesse)**
+* **#Query(Ben)**
+* **#Query(Philipp)**
+* **#Query(Antoine)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* maia (itchysats)
+* others?
+
+## Paper Published!
+
+* [Paper!](https://github.com/Tibo-lg/dlcspecpaper/blob/v0/dlcpaper.pdf)
+  * **#Query(Tibo)**
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* [Numerical Decomposition Clarification](https://github.com/discreetlogcontracts/dlcspecs/pull/182)
+  * Needs Review
+* Using `bigsize` vs. using `uintN`
```

Comment:
That's a conversation I'd like to have for message representation in general. I think we could remove a lot of implementation vulnerabilities with better typing.